css/index.css

```diff
@@ -132,15 +132,107 @@ footer {
 }
 
 footer nav {
-    width: 60%;
+   width: 60%;
     display: flex;
     justify-content: space-between;
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 14px; 
 }
-
 footer nav a {
     color: white;
     text-decoration: none;
+}
+.top-container {
+    width: 800px;
+    margin: 0 auto;
+}
+header a {
+    text-decoration: none;
+    color: black;
+    font-family: 'Rubik', sans-serif;
+}
+header nav {
+    width: 100%;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    font-size: 14px;
+    padding: 20px 2%;
+}
+.main-image {
+    width: 100%;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    font-size: 14px;
+    padding: 20px 2%;
+}
+* {
+    margin: 0px 20px;
+}
```

Comment:
Excellent use of advanced CSS.  You can either use last/first-child as you did or just use nth-child() for all of them.  Any way is fine and you get more experience with selectors by using multiple ones.